CMakeLists.txt

```diff
@@ -189,13 +201,18 @@ if(CONFIG_READABLE_ASM)
   zephyr_cc_option(-fno-partial-inlining)
 endif()
 
-zephyr_cc_option(-fno-asynchronous-unwind-tables)
-zephyr_cc_option(-fno-pie)
+if(NOT ${ZEPHYR_TOOLCHAIN_VARIANT} STREQUAL "xcc")
+  zephyr_cc_option(-fno-asynchronous-unwind-tables)
+  zephyr_cc_option(-fno-pie)
+endif()
```

Comment:
It should be possible to omit the if statement.

Reply:
ok